# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vdr/image-getchar.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- image-0.2.3/libimage/pnm.h.orig	2005-02-14 18:01:00.464704296 +0100
+++ image-0.2.3/libimage/pnm.h	2005-02-14 18:01:11.924962072 +0100
@@ -151,7 +151,7 @@
   Get next significant character, i.e. jump over comments
   @param file - source file
   @param - Next significant character*/
-  bool getchar(FILE* f,char& nNext);
+  bool getchar_(FILE* f,char& nNext);
 
   /*****************************************************************************
   Get next bit from file
--- image-0.2.3/libimage/pnm.c.orig	2005-02-14 18:01:04.294122136 +0100
+++ image-0.2.3/libimage/pnm.c	2005-02-14 18:01:30.060205096 +0100
@@ -73,7 +73,7 @@
 Get next significant character, i.e. jump over comments
 @param file - source file
 @param - Next significant character*/
-bool cPNM::getchar(FILE* f, char& nNext)
+bool cPNM::getchar_(FILE* f, char& nNext)
 {
   register int ich;
   register char ch;
@@ -112,7 +112,7 @@
   register char ch;
   
   do {
-    if(!getchar( f, ch ))
+    if(!getchar_( f, ch ))
       return false;
   } while ( ch == ' ' || ch == '\t' || ch == '\n' || ch == '\r' );
   
@@ -156,7 +156,7 @@
   register T i;
   
   do {
-    if(!getchar( f,ch ))
+    if(!getchar_( f,ch ))
        return false;
   } while ( ch == ' ' || ch == '\t' || ch == '\n' || ch == '\r' );
   
@@ -168,7 +168,7 @@
   i = 0;
   do {
     i = i * 10 + ch - '0';
-    if(!getchar( f,ch ))
+    if(!getchar_( f,ch ))
       return false;
   } while ( ch >= '0' && ch <= '9' );
   nNext = i;
